editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5660,9 +5660,9 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	// those shortcuts one by one.
 	// Resetting zoom to 100% is a duplicate shortcut of `canvas_item_editor/reset_zoom`,
 	// but it ensures both 1 and Ctrl + 0 can be used to reset zoom.
-	ED_SHORTCUT("canvas_item_editor/zoom_3.125_percent", TTR("Zoom to 3.125%"), KEY_MASK_SHIFT | KEY_5);
-	ED_SHORTCUT("canvas_item_editor/zoom_6.25_percent", TTR("Zoom to 6.25%"), KEY_MASK_SHIFT | KEY_4);
-	ED_SHORTCUT("canvas_item_editor/zoom_12.5_percent", TTR("Zoom to 12.5%"), KEY_MASK_SHIFT | KEY_3);
+	ED_SHORTCUT("canvas_item_editor/zoom_3_125_percent", TTR("Zoom to 3.125%"), KEY_MASK_SHIFT | KEY_5);
+	ED_SHORTCUT("canvas_item_editor/zoom_6_25_percent", TTR("Zoom to 6.25%"), KEY_MASK_SHIFT | KEY_4);
+	ED_SHORTCUT("canvas_item_editor/zoom_12_5_percent", TTR("Zoom to 12.5%"), KEY_MASK_SHIFT | KEY_3);
```

Comment:
Shortcut definitions can't have dots/periods in them any more, otherwise we can't tell when it has a feature override,

Reply:
Those shortcut names are pretty awkward anyway, one needs to be pretty math literate to know why `3.125%` is special.

Maybe they could be `zoom_out_x8`, `zoom_out_x16`, `zoom_out_x32`? And same for `zoom_in_` variants.

Reply:
> Maybe they could be zoom_out_x8, zoom_out_x16, zoom_out_x32? And same for zoom_in_ variants.

This sounds better to me indeed.